### ISSUE TITLE
使用另一个数据集训练10轮后accuracy，recall，F1都为0

### ISSUE THREAD

**Issue by cutecharmingkid** [PeiChangY/-BERT-BILSTM-DGCNN-ATTENTION-CRF-#4]
您好，我更换了一个数据集进行训练，即换成了weibo_data,训练10轮后accuracy，recall，F1都为0,我想问一下这种情况正常吗？

建议您看一下我发表在知网上的论文《基于门控空洞卷积神经网络的中文命名实体识别》，里面讲的比较详细，有各个数据集的实体类型

------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "PeiChangY/-BERT-BILSTM-DGCNN-ATTENTION-CRF-"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2023年3月15日(星期三) 下午5:03
***@***.***&gt;;
***@***.***&gt;;
主题:&nbsp;[PeiChangY/-BERT-BILSTM-DGCNN-ATTENTION-CRF-] 关于训练完成后模型效果的提问 (Issue #3)

 
您好，我是使用您的数据集msra_data进行训练，我使用predict文件预测使用的语句是：习近平是中华人民共和国的主席。返回的预测结果为空。同时系统提示：UserWarning: where received a uint8 condition tensor. This behavior is deprecated and will be removed in a future version of PyTorch. Use a boolean condition instead. (Triggered internally at ..\aten\src\ATen\native\TensorCompare.cpp:328.)
 score = torch.where(mask[i].unsqueeze(1), next_score, score)
 
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***&gt;

_Originally posted by @PeiChangY in https://github.com/PeiChangY/-BERT-BILSTM-DGCNN-ATTENTION-CRF-/issues/3#issuecomment-1469748305_

**Comment by we197064** [comment]
> 您好，我更换了一个数据集进行训练，即换成了weibo_data,训练10轮后accuracy，recall，F1都为0,我想问一下这种情况正常吗？
> 
> 建议您看一下我发表在知网上的论文《基于门控空洞卷积神经网络的中文命名实体识别》，里面讲的比较详细，有各个数据集的实体类型
> 
> ------------------ 原始邮件 ------------------ 发件人: "PeiChangY/-BERT-BILSTM-DGCNN-ATTENTION-CRF-" _**@**_._**>; 发送时间: 2023年3月15日(星期三) 下午5:03 _**@**_.**_>; _**@**_.***>; 主题: [PeiChangY/-BERT-BILSTM-DGCNN-ATTENTION-CRF-] 关于训练完成后模型效果的提问 (Issue #3)
> 
> 您好，我是使用您的数据集msra_data进行训练，我使用predict文件预测使用的语句是：习近平是中华人民共和国的主席。返回的预测结果为空。同时系统提示：UserWarning: where received a uint8 condition tensor. This behavior is deprecated and will be removed in a future version of PyTorch. Use a boolean condition instead. (Triggered internally at ..\aten\src\ATen\native\TensorCompare.cpp:328.) score = torch.where(mask[i].unsqueeze(1), next_score, score)
> 
> — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you are subscribed to this thread.Message ID: _**@**_.***>
> 
> _Originally posted by @PeiChangY in [#3 (comment)](https://github.com/PeiChangY/-BERT-BILSTM-DGCNN-ATTENTION-CRF-/issues/3#issuecomment-1469748305)_

最近用自己的数据跑了，结果差不多有90多，还可以，你的还有问题吗？

**Comment by cutecharmingkid** [comment]
> > 您好，我更换了一个数据集进行训练，即换成了weibo_data,训练10轮后accuracy，recall，F1都为0,我想问一下这种情况正常吗？
> > 建议您看一下我发表在知网上的论文《基于门控空洞卷积神经网络的中文命名实体识别》，里面讲的比较详细，有各个数据集的实体类型
> > ------------------ 原始邮件 ------------------ 发件人: "PeiChangY/-BERT-BILSTM-DGCNN-ATTENTION-CRF-" _**@**_._**>; 发送时间: 2023年3月15日(星期三) 下午5:03 _**@**_.**_>; _**@**_.***>; 主题: [PeiChangY/-BERT-BILSTM-DGCNN-ATTENTION-CRF-] 关于训练完成后模型效果的提问 (Issue #3)
> > 您好，我是使用您的数据集msra_data进行训练，我使用predict文件预测使用的语句是：习近平是中华人民共和国的主席。返回的预测结果为空。同时系统提示：UserWarning: where received a uint8 condition tensor. This behavior is deprecated and will be removed in a future version of PyTorch. Use a boolean condition instead. (Triggered internally at ..\aten\src\ATen\native\TensorCompare.cpp:328.) score = torch.where(mask[i].unsqueeze(1), next_score, score)
> > — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you are subscribed to this thread.Message ID: _**@**_.***>
> > _Originally posted by @PeiChangY in [#3 (comment)](https://github.com/PeiChangY/-BERT-BILSTM-DGCNN-ATTENTION-CRF-/issues/3#issuecomment-1469748305)_
> 
> 最近用自己的数据跑了，结果差不多有90多，还可以，你的还有问题吗？

您好，请问您训练了多少轮呀？

**Comment by we197064** [comment]
大概20多轮

**Comment by cutecharmingkid** [comment]
> 大概20多轮

您能留一个邮箱吗？我想有偿向您咨询一些运行项目的问题

**Comment by we197064** [comment]
> > 大概20多轮
> 
> 您能留一个邮箱吗？我想有偿向您咨询一些运行项目的问题

你发个qq号，我加你

**Comment by cutecharmingkid** [comment]
> > > 大概20多轮
> > 
> > 
> > 您能留一个邮箱吗？我想有偿向您咨询一些运行项目的问题
> 
> 你发个qq号，我加你

158916578

**Comment by PeiChangY** [comment]
您好，这里咨询就好，我会尽量回复

**Comment by GitHubwwZ** [comment]
> > > 大概20多轮
> > 
> > 
> > 您能留一个邮箱吗？我想有偿向您咨询一些运行项目的问题
> 
> 你发个qq号，我加你

您好，方便问一下您用自己的数据集跑这个项目的问题吗？我替换成我自己的数据集，跑了20轮三个指标还是都是0.

**Comment by GitHubwwZ** [comment]
> > 您好，我更换了一个数据集进行训练，即换成了weibo_data,训练10轮后accuracy，recall，F1都为0,我想问一下这种情况正常吗？
> > 建议您看一下我发表在知网上的论文《基于门控空洞卷积神经网络的中文命名实体识别》，里面讲的比较详细，有各个数据集的实体类型
> > ------------------ 原始邮件 ------------------ 发件人: "PeiChangY/-BERT-BILSTM-DGCNN-ATTENTION-CRF-" _**@**_._**>; 发送时间: 2023年3月15日(星期三) 下午5:03 _**@**_.**_>; _**@**_.***>; 主题: [PeiChangY/-BERT-BILSTM-DGCNN-ATTENTION-CRF-] 关于训练完成后模型效果的提问 (Issue #3)
> > 您好，我是使用您的数据集msra_data进行训练，我使用predict文件预测使用的语句是：习近平是中华人民共和国的主席。返回的预测结果为空。同时系统提示：UserWarning: where received a uint8 condition tensor. This behavior is deprecated and will be removed in a future version of PyTorch. Use a boolean condition instead. (Triggered internally at ..\aten\src\ATen\native\TensorCompare.cpp:328.) score = torch.where(mask[i].unsqueeze(1), next_score, score)
> > — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you are subscribed to this thread.Message ID: _**@**_.***>
> > _Originally posted by @PeiChangY in [#3 (comment)](https://github.com/PeiChangY/-BERT-BILSTM-DGCNN-ATTENTION-CRF-/issues/3#issuecomment-1469748305)_
> 
> 最近用自己的数据跑了，结果差不多有90多，还可以，你的还有问题吗？

我换成自己的数据集训练，指标都是0，目前跑了20轮，可以问一下你是修改了哪些位置的代码吗？谢谢～
